plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/deserializer/FailoverEndpointDeserializer.java

```diff
@@ -67,7 +68,7 @@ public FailoverEndPoint createNode(IGraphicalEditPart part,AbstractEndpoint obje
 		if(StringUtils.isNotBlank(failoverEndpoint.getName())){
              executeSetValueCommand(END_POINT__END_POINT_NAME, failoverEndpoint.getName());
```

Comment:
Shall we fix this formatting error?

Reply:
Fixed in this PR https://github.com/wso2/devstudio-tooling-esb/pull/272